SingularityUI/app/views/task.coffee

```diff
@@ -9,6 +10,9 @@ class TaskView extends View
             'click [data-action="viewObjectJSON"]': 'viewJson'
             'click [data-action="viewJsonProperty"]': 'viewJsonProperty'
             'click [data-action="remove"]': 'killTask'
+            'submit form#runShell': 'executeCommand'
```

Comment:
for consistency's sake let's stick with using the data-action selectors for all the task action events.